tests/Auth0.ManagementApi.IntegrationTests/RolesTests.cs

```diff
@@ -294,9 +295,32 @@ public async Task Test_paging_includes_totals()
         [Fact]
         public async Task Test_permissions_can_be_retrieved()
         {
-            var userPermissions = await _apiClient.Roles.GetPermissionsAsync("rol_XzVrldh70ox7yjl2", new PaginationInfo(0, 50, true));
+            // Add a new role
+            var newRoleRequest = new RoleCreateRequest
+            {
+                Name = $"{Guid.NewGuid():N}role",
+                Description = $"{Guid.NewGuid():N}description",
+            };
+            var role = await _apiClient.Roles.CreateAsync(newRoleRequest);
+
+            var assignPermissionsRequest = new AssignPermissionsRequest
+            {
+                Permissions = new List<PermissionIdentity>
+                {
+                    new PermissionIdentity { Name = "dotnet:testing", Identifier = "dotnet-testing",  }
+                }
+            };
+            await _apiClient.Roles.AssignPermissionsAsync(role.Id, assignPermissionsRequest);
+
+            // Ensure the Role is assigned to user
+            var userPermissions = await _apiClient.Roles.GetPermissionsAsync(role.Id, new PaginationInfo(0, 50, true));
 
-            Assert.Equal(2, userPermissions.Count);
+            Assert.Equal(1, userPermissions.Count);
+
+
+            // Clean Up
+            await _apiClient.Roles.RemovePermissionsAsync(role.Id, assignPermissionsRequest);
+            await _apiClient.Roles.DeleteAsync(role.Id);
```

Comment:
The test here relied on an existing role for the tenant. I decided to update the test to create, read and delete the role.